File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1983.py
qtd= int(input())
colocado=0
maiorNota=0
for x in range(0,qtd,1):
  matri,nota=map(float, input().split(" "))
  if (float(nota)>=8):
    if(float(nota)>maiorNota):
      maiorNota=nota
      colocado=matri

if maiorNota>=8:
  print("%0.0f"%colocado)  
else:
  print("Minimum note not reached")

File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/1983.js
let qtd=+lines.shift()
const pegarValoress = (line) => line.split(" ").map(a => +a)
let colocado=0
let maiorNota=0
for (let cont =0; cont<qtd; cont++){
        const [matri,nota] = pegarValoress(lines.shift())
        if (nota>=8){
            if(nota >maiorNota){
                maiorNota=nota
                colocado=matri
            }
        }
   }
if (maiorNota>=8){
    console.log(colocado)
}
else if (maiorNota<8){
    console.log("Minimum note not reached")
}